cdc/processor/processor.go

```diff
@@ -387,14 +394,21 @@ func (p *processor) tick(ctx cdcContext.Context, state *orchestrator.ChangefeedR
 	p.pushResolvedTs2Table()
 
 	p.doGCSchemaStorage(ctx)
-	p.metricSyncTableNumGauge.Set(float64(len(p.tables)))
 
 	if err := p.agent.Tick(ctx); err != nil {
 		return errors.Trace(err)
 	}
 	return nil
 }
 
+func (p *processor) memoryConsumption() float64 {
+	var result uint64
+	for _, table := range p.tables {
+		result += table.MemoryConsumption()
```

Comment:
What about histogram? We need per-table memory usage to know if the flow control throttles throughput.